### DISCUSSION TITLE
sqlite - the original, and still holding strong

### DISCUSSION THREAD

**Discussion by yringler** [simolus3/drift#1648] [General]
Was having some performance issues in my app - skipped frames and stuff.
Which is really my laziness - I shouldn't be making 20/30 drift calls on a page, they should be consolidated some how.
So in a moment of weakness, I took a look at the new kid on the block, isar. I think isar is incredible, and do think that one day I'll move to it, but - not sold on stability 100% yet. The development cost in migration, with the risk of having to run back...
I took another look at drift, moved it to an isolate
:boom:
:dash:
Went to super smooth, zero jank.

There is something to be said for super stability and maturity, even in the fast moving tech world. With things working as well as they do now, I might be hanging around for a while longer!

**Comment by davidmartos96** [comment]
Yes, the isolate implementation it's really fast at the moment, even more with the new isolate groups from Dart 2.15. Maybe it should be more visible and recommended to use an isolate, despite that the implementation is a bit more difficult for a beginner.

The lost frames issue is because drift uses FFI, which runs on the same isolate/thread as the UI, so it blocks running animations and such.

**Reply by simolus3** [reply]
I agree that I should probably start recommending isolates more. I made a first step in 6d89a7711b9150edccaf6931c2d912112e439843 by linking the isolate docs from the getting started page :)

**Comment by yringler** [comment]
Just took a look, very helpful and accurate changes 👍
